storage-proofs/core/src/api_version.rs

```diff
@@ -1,5 +1,21 @@
+use anyhow::{self, Result};
+use std::str::FromStr;
+
 #[derive(Debug, Copy, Clone)]
 pub enum APIVersion {
```

Comment:
Lets call this `ApiVersion`, it is a little bit more consistent with naming conventions in rust